test/webdev_proc_utils_test.dart

```diff
@@ -35,9 +37,20 @@ void main() {
       expect(getGlobalWebdevVersion(), isNull);
     });
 
-    test('with webdev activated', () async {
-      await activateWebdev('2.0.0');
-      expect(getGlobalWebdevVersion(), Version.parse('2.0.0'));
+    test('with webdev activated (dart 2)', () async {
+      print(Platform.version);
```

Comment:
Do we intentionally want to print the platform version? or was this left around from a debugging step?

Reply:
That's an oops. I'll remove that.